src/include/partitioning/custom_part_dialogs.rb

```diff
@@ -1674,8 +1674,7 @@ def SubvolHandling(old, new)
             )
             Popup.Message(tmp)
             pth = svtmp + pth
-          end
-          if Builtins.contains(SubvolNames(new), pth)
+          elsif Builtins.contains(SubvolNames(new), pth)
```

Comment:
This PR fixes the original issue but it introduces a new one. If you try to add a `subvolume` which name does not start by *@/*, YaST will [warn the user](https://github.com/alanbortu/yast-storage/blob/4a3e6d1ba74a7b3d5a61daf1a6353f701b12dd0e/src/include/partitioning/custom_part_dialogs.rb#L1667) with this message: *Only subvolume names starting with "@/" currently allowed! Automatically prepending "@/" to name of subvolume.*

Unfortunately, the subvolume won't be created.